newsfragments/2605.bugfix

```diff
@@ -0,0 +1 @@
+``dials.index``: ``joint_indexing=False`` is set automatically for stills.
```

Comment:
Set joint indexing to be automatically defined by default: true for rotation data, false for still shots. This default can be overridden by the user by explicit use of `joint_indexing=`